ai_transform/workflow/context_manager.py

```diff
@@ -45,6 +45,7 @@ def __init__(
         operators: List[abstract_operator.AbstractOperator] = None,
         engine: abstract_engine.AbstractEngine = None,
         metadata: Dict[str, Any] = None,
+        frontend_cta: dict = None
```

Comment:
typing here

Reply:
This is actually a dictionary at this point